rdkcentral/Lightning-UI-Components
# feat(Keyboard): improved keyboard alignment

## Description

In this PR, I have updated changes to align the keyboard properly when the "áöû" key is clicked as per the mocks

Note: I haven't matched the order of _ . - (underscore, dot, hyphen) according to mocks instead matched with previous keyboard alignment when we use abc keys.

## References

LUI-745

## Testing

1. Navigate to Keyboard component and open KeyboardEmail and KeyBoardQuerty
2. Click áöû key and make sure the alignment is matched with mocks
